modules/ossm-tutorial-bookinfo-install.adoc

```diff
@@ -37,12 +37,18 @@ $ oc new-project bookinfo
 
 . Click *Operators* -> *Installed Operators*.
 
-. Click the *Project* menu and choose `openshift-operators` from the list.
+. Click the *Project* menu and choose `bookinfo` from the list.
 
-. Click the *Istio Service Mesh Member Roll* link under *Provided APIs* for the *{ProductName}* Operator.
+. Click the *{ProductName}* Operator.
 
-. Click *Create Service Mesh Member Roll*.
+. Click the *Istio Service Mesh Member Roll* link.
 
+.. If you have already created a Istio Service Mesh Member Roll, click the name, then click the YAML tab to open the YAML editor.
+
+.. If you have not created a Istio Service Mesh Member Roll, click *Create Service Mesh Member Roll*.
++
+NOTE: You need cluster-admin rights to edit the  Istio Service Mesh Member Roll.
```

Comment:
Should this be a proper `[NOTE]` instead?

Reply:
Fixed!